src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -5028,20 +5033,21 @@ public boolean isFileAccessRequestMultiButtonEnabled(){
         }
         return false;
     } 
-    
-    private Boolean downloadButtonAllEnabled = null;
+ 
 
     public boolean isDownloadAllButtonEnabled() {
 
-        if (downloadButtonAllEnabled == null) {
+        boolean downloadButtonAllEnabled = true;
+        
+        if (downloadButtonAllEnabled) { //check to see if it should be false
```

Comment:
This is a bug in current code? isDownloadButtonAllButtonEnabled is always true now.